pkg/maps/lbmap/ipv4.go

```diff
@@ -345,12 +345,12 @@ func (s *Service4Value) GetBackendID() loadbalancer.BackendID {
 	return loadbalancer.BackendID(s.BackendID)
 }
 
-func (s *Service4Value) GetLbAlg() uint8 {
-	return uint8(uint32(s.BackendID) >> 24)
+func (s *Service4Value) GetLbAlg() loadbalancer.SVCLoadBalancingAlgorithm {
```

Comment:
please verify that those functions are properly tested :)

Reply:
Haha, I was already working on it :D

Reply:
Done, PTAL.